CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Changelog
 
+## 0.36.1
+
+**Release date:** 2023-09-18
+
+This prerelease fixes the push branch reported in the logs and status under
+certain circumstances. 
+
+It also upgrades the version of github.com/cyphar/filepath-securejoin that fixes
+[GHSA-6xv5-86q9-7xr8](https://github.com/advisories/GHSA-6xv5-86q9-7xr8). Even though
+the Flux project doesn't consider image-automation-controller to be affected,
+this bump will keep security scanners silent.
```

Comment:
```suggestion
It also upgrades the version of github.com/cyphar/filepath-securejoin that fixes
[GHSA-6xv5-86q9-7xr8](https://github.com/advisories/GHSA-6xv5-86q9-7xr8).
Even though the Flux controllers are not affected by this CVE
since they don't run on Windows nodes,
this bump will keep security scanners silent.
```